src/Germany/Mews.Fiscalizations.Germany/FiskalyClient.cs

```diff
@@ -47,81 +47,76 @@ public async Task<ResponseResult<Model.Client>> CreateClientAsync(AccessToken to
             ).ConfigureAwait(continueOnCapturedContext: false);
```

Comment:
same here. ANd everywhere, I won't go method by method , commenting the same. 😅
But let's wait for the answer I guess.